mdtraj/geometry/distance.py

```diff
@@ -107,7 +107,7 @@ def compute_distances_t(traj, atom_pairs, time_pairs, periodic=True, opt=True):
 
     Returns
     -------
-    distances : np.ndarray, shape=(n_frames, num_atom_pairs), dtype=float
+    distances : np.ndarray, shape=(len(time_pairs), num_atom_pairs), dtype=float
```

Comment:
```suggestion
    distances : np.ndarray, shape=(num_times, num_atom_pairs), dtype=float
```
it was given a name above, right?

Reply:
Oops, my bad, fixed it so they are now consistent.